specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/examples/ConsumerInvitations_RejectInvitation.json

```diff
@@ -0,0 +1,40 @@
+{
+  "parameters": {
+    "location": "East US 2",
+    "api-version": "2020-05-04-preview",
+    "invitation": {
+      "properties": {
+        "invitationId": "dfbbc788-19eb-4607-a5a1-c74181bfff03"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 13 Sep 2017 18:04:32 GMT",
+        "x-ms-request-id": "d5496da4-9c52-402f-b067-83cc9ddea888",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-ratelimit-remaining-tenant-reads": "14999",
+        "x-ms-correlation-request-id": "25c78f97-0b0a-4fe9-ad39-883a482265cd"
+      },
+      "body": {
+        "properties": {
+          "location": "eastus2",
+          "invitationId": "4256e2cf-0f82-4865-961b-12f83333f487",
+          "providerName": "John Smith",
+          "providerEmail": "john.smith@microsoft.com",
+          "providerTenantName": "microsoft",
```

Comment:
>providerTenantName [](start = 11, length = 18)

names could be spoofed, should this not show the tenantID?

Reply:
We take the names directly from graph, Not sure if tenant ids will provide better trust for customers, in most cases tenant ids are not publicly known.